CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## [0.11.2] - 2020-10-29
+
+### What's New
+ - Added rate limiting to logging that occurs under certain misconfigurations (#1901)
+
+### What's Changed
+- Fixed an issue in the Upstream Authority plugin that could result in stale data being used (#1917)
```

Comment:
My personal preference for these kinds of changelog lines is to state things in terms of what the user would be experiencing if they were encountering this issue. That way, people can read the changelog and think "oh yea I remember seeing something like that". For example:
```suggestion
- Fixed an issue in the Upstream Authority plugin that could result in a delay in the propagation of bundle updates/changes (#1917)
```

Reply:
much better, thanks!